enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx

```diff
@@ -62,7 +62,9 @@ export const InteractiveQuestionProvider = ({
     isError: isCardIdError,
   } = useValidatedEntityId({
     type: "card",
-    id: initialQuestionId,
+
+    // If the question is new, we won't have a question id yet.
+    id: initialQuestionId === "new" ? undefined : initialQuestionId,
```

Comment:
`useValidatedEntityId` expects `undefined` in case of no card ids, so we have to cast it.